service/history/task/transfer_active_task_executor_test.go

```diff
@@ -876,8 +875,8 @@ func (s *transferActiveTaskExecutorSuite) expectCancelRequest(childDomainName st
 			s.Equal(childDomainID, request.DomainUUID)
 			s.Equal(childDomainName, request.CancelRequest.Domain)
 			s.True(request.GetChildWorkflowOnly())
-			errors := []error{nil, &types.CancellationAlreadyRequestedError{}, &types.EntityNotExistsError{}}
-			return errors[rand.Intn(len(errors))]
+			errs := []error{nil, &types.CancellationAlreadyRequestedError{}, &types.EntityNotExistsError{}}
+			return errs[rand.Intn(len(errs))]
```

Comment:
I think this is a bad change - lets not do that.

Reply:
@jakobht This change only rename the local variable errors because it collides with the imported package errors.